### PR TITLE
Fix/social media handle

### PR DESCRIPTION
**Pull request by RB-Isiaq**
> Added the social media handle key to the user payload

### CODE REVIEW

File: src/components/ApplicationSteps/Step5Application/Internals/EmailVerified/EmailVerified.jsx [hunk 1]
```diff
@@ -46,7 +47,9 @@ export const EmailVerified = () => {
 
         dispatch(emailVerification(true));
         dispatch(storeUser(decode));
-        // nav('/welcome');
+        setTimeout(() => {
```
Comment (edustipend-root) [inline src/components/ApplicationSteps/Step5Application/Internals/EmailVerified/EmailVerified.jsx:50]: What does this change do?

File: src/components/ApplicationSteps/Step5Application/Internals/Submit/Submit.jsx [hunk 2]
```diff
@@ -30,7 +30,10 @@ export const Submit = () => {
     reasonForRequest: reasonForRequest,
     stepsTakenToEaseProblem: stepsTakenToEaseProblem,
     potentialBenefits: potentialBenefits,
-    futureHelpFromUser: futureHelpFromUser
+    futureHelpFromUser: futureHelpFromUser,
+    socialMediaHandles: {
+      twitter: twitterHandle
```
Comment (edustipend-root) [inline src/components/ApplicationSteps/Step5Application/Internals/Submit/Submit.jsx:35]: @RB-Isiaq, we should be sending whatever the user selects. If they picked Facebook this should be `facebook` not `twitter`, correct?

**Review by edustipend-root — CHANGES_REQUESTED** [review]
Update the social media handle to reflect whatever the user selects.

File: src/components/ApplicationSteps/Step5Application/Internals/Submit/Submit.jsx (hunk 2 above)
Comment (RB-Isiaq) [reply src/components/ApplicationSteps/Step5Application/Internals/Submit/Submit.jsx:35]: image
We actually have this on the frontend, it's only a twitter handle field

File: src/components/ApplicationSteps/Step5Application/Internals/EmailVerified/EmailVerified.jsx (hunk 1 above)
Comment (RB-Isiaq) [reply src/components/ApplicationSteps/Step5Application/Internals/EmailVerified/EmailVerified.jsx:50]: After a successful email verification, the emailVerification state is changed to true, and the user state is set to the decoded information from the token, thereby storing and persisting the userInfo to localstorage.

The nav(0) refreshes the page, and navigates the user to the welcome page, if authenticated.

File: src/components/ApplicationSteps/Step5Application/Internals/Submit/Submit.jsx (hunk 2 above)
Comment (RB-Isiaq) [reply src/components/ApplicationSteps/Step5Application/Internals/Submit/Submit.jsx:35]: image

Comment (RB-Isiaq) [reply src/components/ApplicationSteps/Step5Application/Internals/Submit/Submit.jsx:35]: Made some changes, to allow users select and insert their handles

Screenshot 2024-01-14 075641
Screenshot 2024-01-14 075717
Screenshot 2024-01-14 080414

Comment (edustipend-root) [reply src/components/ApplicationSteps/Step5Application/Internals/Submit/Submit.jsx]: Thanks for making this change @RB-Isiaq

**Pull request merged**